crates/engine/tree/src/tree/payload_processor/multiproof.rs

```diff
@@ -310,15 +283,15 @@ impl StorageMultiproofInput {
     }
 }
 
-/// Input parameters for spawning a multiproof calculation.
+/// Input parameters for dispatching a multiproof calculation.
 #[derive(Debug)]
 struct MultiproofInput {
     config: MultiProofConfig,
     source: Option<StateChangeSource>,
     hashed_state_update: HashedPostState,
     proof_targets: MultiProofTargets,
     proof_sequence_number: u64,
-    state_root_message_sender: Sender<MultiProofMessage>,
+    state_root_message_sender: CrossbeamSender<MultiProofMessage>,
```

Comment:
now it's only used for `EmptyProof` message, I wonder if we can remove it altogether and just send the empty proof to `proof_result_tx`? No need to address it in this PR, just a thought cc @mattsse